scripts/tools_assure_gogenerate.sh

```diff
@@ -4,7 +4,8 @@ go get -u golang.org/x/tools/cmd/stringer github.com/tinylib/msgp
 cd $GOPATH/src/github.com/raintank/metrictank/
 # stringer needs all packages to be installed to work
 # see https://github.com/golang/go/issues/10249
-go install ./...
+#go install ./...
+go install github.com/raintank/metrictank/vendor/github.com/hashicorp/memberlist github.com/raintank/metrictank/vendor/github.com/dgryski/go-tsz 
```

Comment:
i think the packages need to be installed as "github.com/hashicorp/memberlist" and not "github.com/raintank/metrictank/vendor/github.com/hashicorp/memberlist"

i think you can do that by running
```
govendor go install github.com/hashicorp/memberlist
```